source/features/mark-merge-commits-in-list.tsx

```diff
@@ -37,8 +37,7 @@ const filterMergeCommits = async (commits: string[]): Promise<string[]> => {
 
 function getCommitLink(commit: HTMLElement): HTMLAnchorElement | undefined {
 	return $([
-		'a.markdown-title', // Old view style (before November 2023)
-		'.markdown-title a',
+		'[data-testid="list-view-item-metadata-item"]:last-child a',
 	], commit);
```

Comment:
Since there's only one selector we can remove the array. Also I wonder if there's a way to more precisely select the commit element there, for example by targeting the copy button and reading its attributes